packages/apps-config/src/settings/ethereumNetworks.ts

```diff
@@ -1,4 +1,4 @@
 // Copyright 2017-2020 @polkadot/apps-config authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-export const ethereumNetworks = ['node-moonbeam', 'moonbeam-alphanet', 'moonbase-alpha', 'moonbeam-standalone'];
+export const ethereumNetworks = ['node-moonbeam', 'moonbase-alphanet', 'moonbeam-standalone'];
```

Comment:
It needs `moonbeam-alphanet`, this is what the state.getRuntimeVersion RPC returns -

```
1: state.getRuntimeVersion
{
  specName: moonbeam-alphanet,
  implName: moonbeam-alphanet,
  authoringVersion: 2,
...
```

Reply:
interesting because I get moonbase-alphanet from the stagenet
let me add both then

Reply:
pushed